BrownField/Auto-assessment/scripts/Main.ps1

```diff
@@ -8,23 +8,23 @@
 function Main {
     try {
         
-        $tenantId = ""
-        $subscriptionId = ""
+        $tenantId = "27eda52d-06a5-4e9f-bd76-1a062e47aba0"
+        $subscriptionId = "d52f9c4a-5468-47ec-9641-da4ef1916bb5"
         
         # Provide the names of the SDDCs to test
         # If the array is empty, all SDDCs in the subscriptio will be tested
         # Example: $namesofSddcsToTest = @()
         # If you want to test specific SDDCs, provide the names in comma-separated format
         # Example: $namesofSddcsToTest = @("prod-sddc", "uat-sddc")
-        $namesofSddcsToTest = @()
+        $namesofSddcsToTest = @("CHA-POC-AVS")
```

Comment:
@Mahesh-MSFT , similarly, I believe this hardcoded value to the parameter can be removed.

Reply:
Done.